sources/programs/even-odd/php/how-to-implement-the-solution.md

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
This text will actually appear after the code sample. It is supposed to describe how the sample works. Please see the [existing article](https://sampleprograms.io/projects/even-odd/php/). This is the code you would be writing about.